lib/Graph/Graph.cpp

```diff
@@ -959,25 +959,30 @@ ARITHMETIC_FUN_DEF(Max);
 ARITHMETIC_FUN_DEF(Min);
 #undef ARITHMETIC_FUN_DEF
 
+/// \returns the result type for a logical operation.
```

Comment:
I am not sure what you mean buy "logical op".  When I read the code it looks like it's uniquing the type, which is something that that name does not specify. I am not sure what's going on here.

Reply:
This helper function is used only by the functions creating logical operations like `createCmpEQ` and `createCmpLTE` to compute their result types. Hence the name. The result type is computed based on the type of the operand of `createCmpEQ`. In quantized cases we provide concrerte scale and offset for the type as we know that the result of a logical operation could be only 0 or 1. I can extend the function comment with this information.